requirements_dev.txt

```diff
@@ -1,8 +1,8 @@
-PyYAML==3.11
+PyYAML==3.13
 Sphinx==1.3.1
 bumpversion==0.5.3
 coverage==4.0
-cryptography==1.0.1
+cryptography==2.7
```

Comment:
no idea why this is needed, but hey, tests pass and it doesn't look directly used by the library.  

otherwise,this won't install due to not supporting libressl version on the docker image that I found. similar issues occur with other python images, but this seemed the best one that could still use the original project's tox config